### ISSUE TITLE
Добавить возможность опозновать неизвестные устройства для загрузчика

### ISSUE THREAD

**Issue by Roundabout1** [kruzhok-team/lapki-client#453]
В Arduino IDE присутствует возможность опознать неизвестные устройства через интерфейс (нужно указать порт к которому подключено устройство и её тип), предлагаю реализовать что-то похожее для Lapki-IDE.

**Comment by chekoopa** [comment]
Согласен, но лучше прятать это за ключ и по умолчанию включать только для локального загрузчика. Больно заманчивая дыра в системе получится.
